docs/docs/tutorial/chapter1/prerequisites.md

```diff
@@ -17,11 +17,11 @@ You could definitely learn them all at once, but it will be harder to determine
 
 ### Redwood Versions
 
-You will want to be on at least version 1.0.0 to complete the tutorial. If this is your first time using Redwood then no worries: the latest version will be installed automatically when you create your app skeleton!
+You will want to be on at least version 5.0.0 to complete the tutorial. If this is your first time using Redwood then no worries: the latest version will be installed automatically when you create your app skeleton!
```

Comment:
This is probably the most controversial change. I feel like "at least v1" probably isn't true anymore. Thoughts?